cellxgene_schema_cli/tests/test_validate.py

```diff
@@ -281,8 +281,10 @@ def test__validate_with_h5ad_invalid_and_without_labels(self):
 
 
 class TestSeuratConvertibility(unittest.TestCase):
-    def validation_helper(self, matrix):
+    def validation_helper(self, matrix, raw=None):
         data = anndata.AnnData(X=matrix, obs=good_obs, uns=good_uns, obsm=good_obsm, var=good_var)
+        if raw:
+            data.raw = raw
```

Comment:
ah, I see. I guess AnnData allows this. The class prevented me from mutating the `.raw` attr _on_ the AnnData object itself.